.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 0 * * *"
+
+concurrency:
+  group: build-${{ format('{0}-{1}', github.head_ref || github.run_number, github.job) }}
+  cancel-in-progress: true
+
+env:
+  BUILDKIT_PROGRESS: plain
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

Comment:
The latest LTS is 24.04. Is there any reason for running 20.04 in this runner?

Reply:
I changed this to `ubuntu-latest`